jest.config.ts

```diff
@@ -10,6 +10,7 @@ const cfg: Config = {
   coverageReporters: ["json", "lcov", "text", "clover", "json-summary"],
   reporters: ["default", "jest-junit", "jest-md-dashboard"],
   coverageDirectory: "coverage",
+  testTimeout: 10000,
```

Comment:
Is it because of slow network requests?

Reply:
[This](https://github.com/rndquu/conversation-rewards/blob/9ce0843236d6592824f6e0f0f5ecc95c20c2268f/tests/process.issue.test.ts#L222) test sometimes takes more than 5 seconds hence CI was [failing](https://github.com/ubiquibot/conversation-rewards/actions/runs/9774430781/job/26983496760) in some cases